doc-templates/UpdaterConfig.md

```diff
@@ -11,25 +11,23 @@
 This section covers all options that can be set in the *router-config.json* in the 
 [updaters](RouterConfiguration.md) section.
 
+Real-time data are those that are not added to OTP during the graph build phase but during runtime.
 
-## Real-time data
+Real-time data sources are configured in the `updaters` section is an array of JSON objects, each
+of which has a `type` field and other configuration fields specific to that type. 
+
+## GTFS-Realtime
 
 GTFS feeds contain *schedule* data that is published by an agency or operator in advance. The
 feed does not account for unexpected service changes or traffic disruptions that occur from day to
 day. Thus, this kind of data is also referred to as 'static' data or 'theoretical' arrival and
 departure times.
 
-Real-time data sources are configured in the `updaters` section is an array of JSON objects, each
-of which has a `type` field and other configuration fields specific to that type. Common to all
-updater entries that connect to a network resource is the `url` field.
-
-### GTFS-Realtime
-
 The [GTFS-RT spec](https://developers.google.com/transit/gtfs-realtime/) complements GTFS with three
 additional kinds of feeds. In contrast to the base GTFS schedule feed, they provide *real-time*
 updates (*'dynamic'* data) and are updated from minute to minute.
 
-### Configuring real-time updaters
+#### Types of data delivery
```

Comment:
Shouldn't this be a higher level header, than the individual updater types?

Reply:
Perhaps we can get rid of this paragraph alltogether. I feel that this is from a time where the structure of this document was rather different. WDYT?

Reply:
If we feel that we need this information we can provide it in each section.

Besides, there doesn't seem to be any documentation for HSL's MQTT updater.

Reply:
I think we need a information hierarchy on the page. It is a bit complex, since SIRI and GTFS-RT can produce both trip updates and alerts, so we need to decide which to have on a higher priority.

Reply:
That looks much better